### ISSUE TITLE
Bow/crossbow strength requirements are unrealistic

### ISSUE THREAD

**Issue by chaisawlajatang** [CleverRaven/Cataclysm-DDA#38642]
# Describe the bug

The recent archery update seriously brought up both damage and strength requirements for bows and crossbows. I dunno about damage, buth the STR requirements seem very unrealistic. I keep in mind design values - an average human has 8 strength, a strong human has 10, a very strong person has 12, a top-tier athlete has 14. Look:

1. A 205 crossbow/200 lb composite crossbow requires 12 strength to draw, compound crossbow requires 11. [Here](https://www.youtube.com/watch?v=2IdfmaC_t-Q) we have a man, drawing about 150 lbs with his bare hands, 300 lbs in sitting span and even more with very basic devices (**1500** lbs with a windlass, so the ingame heavy crossbow can be easily upped from the current 800 lbs). He's really not looking like a world-class athlete for me.
2. [Here's](https://www.youtube.com/watch?v=2IdfmaC_t-Q) another average man drawing crossbows. Yes, he's using basic devices - goats foot lever, the spanning belt and the gaff lever, but they're like REALLY basic ones - every survivor should be able to use one. He claims you're able to draw about **450 lbs** this way.
3. [Here's](https://www.youtube.com/watch?v=ohutwHqClOc) the guy drawing 80 lbs bows. 80 lb in-game bow requires 12 (!) strength, and the guy from the video is seriously not looking like the athlete or heavylifter.
4. [Here's](https://www.youtube.com/watch?v=CoSCl_qcJsQ) a man with 110 lb bow (14 STR req). Yeah, he's definitely strong, but he's not a world-class powerlifter.
5. [Here's](https://www.youtube.com/watch?v=-KokNtdh4_Y) a regular man easily drawing a 60 lb bow (10 STR req). Is he really about a half-way to the olympic championship?
6. [Here's](https://www.youtube.com/watch?v=MjY2QrU4sm4) a strong man indeed drawing 210 lb bow. Ingame wooden 145 lb greatbow requires 19 (!) STR. Man from the video is not a cyborg or mutant, I'm sure.
7. [Here's](https://www.youtube.com/watch?v=xrzPFIjAZdU) a 100 lb bow. 
8. [Here's](https://www.youtube.com/watch?v=WmhYwboqBOg) a 90 lb one. 

There are lots of evidences, I think you got the point - both bows and crossbows, despite the great drawing weight, do not require tremendous physical strength as they require now.

# Steps To Reproduce

1. Start as an average person with 8-9 strength.
2. Spawn a compound bow (high), composite bow, reflex bow, hybrid longbow, crossbow or composite crossbow.
3. There's no way to shoot with them.

# Expected behavior

Able to shoot from a bow or crossbow. A bow/crossbow with req 12 or even 14 STR is probably a ballista or similar siege machine. I don't touch damage or range, because they are calculated by some complex formulas, but STR req is set arbitrary, so it can be changed more easily.

The lower cap is currently 30 lbs with 10 lb/1 STR. I suppose to remove the cap - straight 10lb/1 STR would be fine: i.e. 6 STR person could draw a string of a 60-lb hunting bow, but stronger person would still benefit in bonus range. 8 STR person could draw a 80-lb bow, etc. 

Self bow - 30 lbs - I guess even a feeble person or a kid can draw 30 lbs.
Short bow - 50 lbs - maybe leave it as is at 6 STR, I guess, or lower to 5.
Compound bow (low) - 40 lbs - it seems even a feeble person is able to draw it.
Compound bow (normal) - 60 lbs - lower the requirement to 6 STR.
Compound bow (high) - 80 lbs - to 8 STR. Compound bows are very easy to draw thanks to a pulley system.
Composite bow - 110 lbs - to 11 STR.
Recurve bow/Reflex bow - 60 lbs - to 6 STR.
Longbow - 80 lbs - to 8 STR.
Wooden greatbow - 145 lbs - to 14 STR. Maybe 15 if it strictly designed for superhumans only.
Compound greatbow - 120 lbs - to 12 STR.
Reflex recurve bow - 50 lbs - to 5 STR.

8 STR max for all crossbows (and even without the req at all with drawing devices).

TL;DR: 8-9 STR should be more than enough for most string weapons, more for the heaviest ones only.

**Comment by Tamiore** [comment]
Crossbows don't HAVE a minimal strength requirements last I've checked (strength effects reload speed). 
Strength 14 IS effective an almost siege engine, over bows can be adjusted for strength. 
What exactly is your problem, again?

**Comment by chaisawlajatang** [comment]
>Crossbows don't HAVE a minimal strength requirements

They do.
![image](https://user-images.githubusercontent.com/40278862/76166193-c7d25000-616d-11ea-886d-acf9df861aaa.png)

**Comment by Tamiore** [comment]
>they do. 

We need to find PR that introduced it, then, and roll it back. 
It was either a mistake or an oversight.

**Comment by chaisawlajatang** [comment]
>We need to find PR that introduced it, then, and roll it back.
>It was either a mistake or an oversight.

Ok, [here it is.](https://github.com/CleverRaven/Cataclysm-DDA/pull/37241)

**Comment by Tamiore** [comment]
I see no mention of minimal strength required for crossbows being introduced. 
So, yea, pretty sure it's an oversight.

**Comment by Tamiore** [comment]
> Strength 14 bow, as declared, has 90 lb draw weight. This is the 100 lb IRL bow, this is 90 lb - do they look like siege weapons?

So the problem is inconsistent description?
You want str 14 bow to list higher draw weight?

**Comment by chaisawlajatang** [comment]
>So the problem is inconsistent description?

The problem is inconsistency between description, strength requirement and damage. A weapon shoud either deal more damage and/or have a description of a hand-wielded siege ballista (and more common realistic weapons be in place), either have strength requirements and/or damage lowered.

**Comment by Tamiore** [comment]
> The problem is inconsistency between description, strength requirement and damage. A weapon shoud either deal more damage and/or have a description of a hand-wielded siege ballista (and more common realistic weapons be in place), either have strength requirements and/or damage lowered

Ok, so what draw weight should it have in it's description to be consistent with the game stats?

**Comment by davidpwbrown** [comment]
In the linked PR , Kevin stated "There's no coherent case for crossbow poundage limits, so these are all set at 8 when the style of crossbow calls for it, and the limit is absent in cases when there is some built-in mechanical advantage."

If there are crossbow strength requirements higher than 8, then thats an oversight, from what I can gather.

**Comment by chaisawlajatang** [comment]
>Ok, so what draw weight should it have in it's description to be consistent with the game stats?

My main concern is NOT the description.
Looks like the numbers were a bit arbitrary

>A somewhat more arbitrary number is the mapping of strength stat to weapon poundage.
The document that Darktoes put together perpetuates the assumption that each point of strength stat is good for 10 pounds of draw weight on a bow. This doesn't make much sense, as it places the low end of bows at 30lbs within the reach of an incredibly feeble str 3 character, and the main "used by fit hunters" 60lb draw bows in reach of a still significantly-weaker-than-average str 6 character. AFAICT the goal here was to keep the mapping trivial and to have that simple mapping span all the way to nonsensical 200lb draw weight bows. I've instead opted to try and place the 30lb draw at the low end of strength but not utterly feeble, and to place the 60 lb draw at a bit over average, and then the very high draw weight bows land in the upper limits of strength.

So I suppose 10lb/1 STR would be fine: i.e. 6 STR person could draw a string of a 60-lb hunting bow, but stronger person would still benefit in bonus range. 8 STR person could draw a 80-lb bow, etc.

**Comment by Tamiore** [comment]
> My main concern is not the description.

Ok, but what *is* it, then?
What exactly to you propose to change about the 14 str bow?

> advantage."

>If there are crossbow strength requirements higher than 8, then thats an oversight, from what I can gather 

The screenshot seems to show a requirement to *shot* the crossbow, not to reload it. That does not make *any* sense.

**Comment by chaisawlajatang** [comment]
>Ok, but what is it, then?

Sure, I'll copy it again.

>The problem is inconsistency between description, _strength requirement and damage_. A weapon shoud either deal more damage and/or have a description of a hand-wielded siege ballista (and more common realistic weapons be in place), either have strength requirements and/or damage lowered.

I'll rephrase it - the bows require too much strength for their draw weight.

>What exactly to you propose to change about the 14 str bow?

I propose to change STR requirements: make them 8 for all crossbows and set STR requirements for bows in ratio 1 STR=10 lb draw.

**Comment by Tamiore** [comment]
> I'll rephrase it - the bows require too much strength for their draw weight.

Bows in the game don't technically have a "draw weight" in any sense other than whatever is written in their description. Damage is not really a strict function of draw weight beyond  "the higher the draw weight, the higher the damage". 
So changing the draw weight for the bow with the highest draw weight is effectively just changing the description. Unless you want to change the damage * as well*.

**Comment by chaisawlajatang** [comment]
>Bows in the game don't technically have a "draw weight" in any sense other than whatever is written in their description

But draw weight is tied to STR requirement in ratio I posted above.

>So changing the draw weight

My main proposal is not changing the draw weight, but rather change STR requirements to be lower. 

>Unless you want to change the damage as well.

Damage is not really a strict function of draw weight, there's apparently a lot of complex calculations made not tied to draw strength.

>The rationale for the damage numbers is that we make some assumptions about weapon draw weight and efficiency based on the understanding of what real-world item each weapon represents, and once we have an estimated power output, we take the square root of the presented projectile momentum as the base damage number. Then this number is modified by the terminal ballistics characteristics of the arrow, mostly the head, to determine the final damage number. In practice this falls into a handful of categories.

**Comment by Tamiore** [comment]
Ok, so I went ahead and checked the **latest** experimental.  

What … the … hell?

Compound crossbow has TWICE the damage of heavy crossbow with 1/5 the reload time.
Compound bow has damage BELOW compound GREAT-bow. 
Etc, etc?
Huh? What's going on?

**Comment by jeremyshannon** [comment]
What's going on is the archery system is being reworked, and it's not done. So now is a good time to talk about it, and for him to bring up his observations -- if you don't need to be a world-class strongman/Olympic athlete just to draw that composite bow from the sporting goods store, then 14 str is definitely too high a minimum requirement for that.

**Comment by Tamiore** [comment]
> then 14 str is definitely too high a minimum requirement for that.

What bow are you talking about?
Not bow with min str 14 requirements exists in latest experimental.

**Comment by UndeadMoose** [comment]
![Screenshot (649)](https://user-images.githubusercontent.com/56947231/76171160-94bea980-61cb-11ea-8a59-997175c4b870.png)

**Comment by Tamiore** [comment]
Ok, that helps. 
These stats are part of a very recent change. 
Whoever made that PR really is the only one who can explain whatever is that bow supposed to be. 
Because to me it's not making much sense. 
It's an auto-learn bow made with a stone knife from a few sticks and bones that has damage on part with assault rifle.

**Comment by Zourin2** [comment]
The problem has occurred largely in disproportion to the fact that now 14 strength is the upper boundary for human strength.  The average human is more than capable of using any bow except maybe the compound greatbow, which exists for the purpose of advanced strength individuals. 

Even slightly weaker individuals are able to use competitive bows, as the archery skill itself can easily account for the specific muscle groups the activity trains without having to turn the character into an olympian.

**Comment by FuelType-Memes** [comment]
I have watched the videos that you've linked and there is a great point Joe Gibbs, archer with a 200+ pound bow, makes: modern techniques are for modern bows, traditional warbows require traditional technique. When he draws a bow he uses muscles that aren't used that much in modern archery. Thus, I suggest, that STR requirement should depend on the archery skill, because even though you can be very strong, your lack of technique would prevent you from applying your strength effectively

**Comment by jeremyshannon** [comment]
In archery.json as of yesterday, the composite bow read:` "min_strength": 14` Back on Mar 5 it was 16 for some reason.

**Comment by AncientSion** [comment]
I use a longbow in reallife with 60 pounds and i can draw it just fine. I would say im strength 9, So 10 is certainly not right. Probably 8.

**Comment by Zourin2** [comment]
Compound bows and crossbows are designed to substantially reduce draw weights, because this dude Archimedes figured out how pulleys work. The same pulleys on compound bows and crossbows. A 200lb compound bow only requires 50lbs of draw weight because, surprise, two pulleys.

**Comment by kevingranade** [comment]
Thanks for the sources!
The Tod's workshop video is nicely informative, he claims 150lbs is doable for hand spanning (no stirrup, no other tools), so we can take that and call it doable for a strength 8 "average strength person".  I'll take a look at bumping the basic crossbow to 150lbs, requiring 8 strength and adjusting damage commensurately.  Sitting spanning and use of a stirrup will need to wait as this crossbow is still stuck at incredibly low requirements for construction.  Alternately I could stick a stirrup on it, bump it back to 200lbs, and ditch the (actually terrible) crafting recipe.  Something similar applies to the remainder of the crossbows except the pistol crossbow and the Gantz-Rustang (which I'll be renaming to Arbalest, but not until after 0.E).

Someone can mess with adding a seated spanning feature and then we can revisit weights again.  Luckily the outcome of all my research was that I can map from poundage to momentum to damage in a pretty straightforward way, so I don't have to redo a ton of work when we tweak this stuff.

Everything about spanning devices is irrelevant, until someone adds support for using them, we're going to pretend they don't exist.  The exception is the windlass crossbow where it's baked into the item.

"This guy (that's clearly not a bodybuilder) can do X":
You've stumbled across a shortcoming of the stat system, which is that a character with X strength has uniform fitness across their entire body. It's incredibly common in various sports for someone to overdevelop specific muscles without "looking like a bodybuilder".  You can develop an incredibly strong set of back muscles without it being visible and draw a stronger bow than it "looks like you should be able to".  As you can see with the guy drawing the 210 lb bow, he intensively trained for this, at the end of which his muscles *that he's using for drawing the bow* are world-class.

What that means for the game is that in order to do *any* feat of strength, you have to "pay for" having that level of strength across your entire body.  I'm not against revisiting that somehow in the future, but until we do, your character is going to need to pay for a certain level of strength in order to use it.  That means that yes, you need to have above average strength to shoot a bow with above average draw weight.

"A bow/crossbow with req 12 or even 14 STR is probably a ballista or similar siege machine."
No, it's not.  There is a fairly well known range of usable bow and crossbow weights, and long before you have something you would call a ballisa or siege engine, the strength requirement has gone away entirely because there's no possible way for any human to draw/span them without massive mechanical advantage.  That 1,500 lb arbalest? *still not a ballisa*.

The highest weight *your source* calls out as *achievable* for spanning a crossbow by hand is the 380 lb (and I'm assuming they mean with a sitting span or stirrup or spanning belt).  So maybe we can stick a pin in that and revisit it later, but I don't see any reason to have crossbows above the 250 lbs you get with a stirrup and an "average shooter" except the arbalest which comes with a windlass.

In short, I'll concede your point and stick all crossbow strength requirements at 8, since based on your sources (Tod's workshop, the rest don't help), that yields a usable amount of draw strength and therefore damage.

Now bows are another thing. Basically your sources don't make the case you seem to think they make, and your other rationales don't hold up either. In particular, "Compound bows are very easy to draw thanks to a pulley system." is wrong, the rating of a compound bow or crossbow is based on the *effective* draw weight *at the bowstring*, not anything to do with the limbs.  In other words, if you have a 80lb draw, you are still going to need to pull with 80lbs of force at some point in the draw.

In other other words, you need to be just as strong to draw an 80lb compound bow as a 80lb straight bow.  I don't see anything that makes me inclined to revisit the existing bow draw strengths.

**Comment by Zourin2** [comment]
"You've stumbled across a shortcoming of the stat system, which is that a character with X strength has uniform fitness across their entire body."

**Comment by Zourin2** [comment]
"Power
You’ll find that the top end of bows on the market for both styles coming in at around 70 lbs of draw weight. 70 lbs is enough power to accomplish pretty much anything anyone wants to accomplish with a bow. Manufacturers and retailers don’t tend to stock 80 lbs bows for this reason, these isn’t much call for them.

In an ideal world, with ideal conditions and ideal measurement a 70 lbs draw weight compound bow should perform better than a 70 lbs recurve drawn to an optimum length and using exactly the same length and weight of arrow. You’d think they should both impart the same force but in reality the compound system of string and cams is more efficient at flinging arrows and ends up being more powerful in the field"

https://targetcrazy.com/archery/resources/compound-vs-recurve/

**Comment by FuelType-Memes** [comment]
I must reiterate that archery skill should represent your body becoming more comfortable with drawing the bow. I would even go as far as to say that when you have low archery skill it should increase the required strength for bows because you lack the proper technique that makes drawing and shooting easier

**Comment by Zourin2** [comment]
On crossbows:
"The principle specification that distinguishes a hunting crossbow from a target model is draw weight. Fortunately, most states have taken the guesswork out of determining what is sufficient by establishing a regulatory minimum. While those regulations vary considerably from state to state, the overall range runs from 75 to 125 pounds of draw weight.

With little exception, any crossbow in that range should be adequate to kill a whitetail deer at moderate ranges. That said, most of the better hunting crossbows are in the 150 to 175 pounds range, with a few topping out over 200 pounds. In general, bigger is better—or in this case, faster. Keep in mind that you can go too heavy, too, as at least one state—Ohio—has a maximum allowable draw weight of 200 pounds."

**Comment by Tamiore** [comment]
To stress the obvious: while a minimal strength (of 8, or whatever) might be warranted requirement to reload a crossbow without any aid, that's what it should limit: reloading. Not shooting it once it's already reloaded.

**Comment by JoeFreeman-Adv** [comment]
Just a little tidbit:

In his newest video Joerg Sprave (the slingshot channel guy) modifies a 110 pound warbow with his instant legolas device.
While the video and the device itself is pretty interesting the thing that really caught my attention is how hard it is for him to use the bow on it's own.
It shows the difference of having general upper body strength and the technique and muscles needed for firing such a weapon.

https://youtu.be/Ovl6_UEIWGY

**Comment by Zourin2** [comment]
> Just a little tidbit:
> 
> In his newest video Joerg Sprave (the slingshot channel guy) modifies a 110 pound warbow with his instant legolas device.
> While the video and the device itself is pretty interesting the thing that really caught my attention is how hard it is for him to use the bow on it's own.
> It shows the difference of having general upper body strength and the technique and muscles needed for firing such a weapon.
> 
> https://youtu.be/Ovl6_UEIWGY

Also, to note, a 100lb war bow is also an outlier, as half that is closer to a serviceable average for hunting and marksmanship. Nobody's going to *have a book on bow making* then make a bow and be like "I've never shot a bow before, lets make a 100lb composite recurve bow".

**Comment by AncientSion** [comment]
Make a boy that is usable str 8 then.
Self bow is 6
Short bow is 9.

What about the AVERAGE guy at str 8 ?

**Comment by kevingranade** [comment]
See #38657 4 bows are now str 8 or lower, and three more are 9 or lower.
